Doc/library/os.rst

```diff
@@ -1180,8 +1180,27 @@ as internal buffering of data.
    respectively.
 
    .. versionadded:: 3.3
-      Some operating systems could support additional values, like
-      :const:`os.SEEK_HOLE` or :const:`os.SEEK_DATA`.
+
+.. data:: SEEK_HOLE
+          SEEK_DATA
+
+   Parameters to the :func:`lseek` function and the :meth:`~io.IOBase.seek`
+   method on file objects, for seeking file data and holes.
+
+   :data:`!SEEK_DATA`
+      Adjust the file offset to the next location in the file containing data.
+
+   :data:`!SEEK_HOLE`
+      Adjust the file offset to the next hole in the file.
+      A hole is defined as a sequence of zeros.
+
+   .. note::
+
+      These operations only make sense for filesystems that support them.
+
+   .. availability:: Linux >= 3.1, macOS, Unix
```

Comment:
Well, macOS is Unix, so perhaps this:

```suggestion
   .. availability:: Linux >= 3.1, Unix
```

Reply:
I think for the casual reader having macOS called out specifically is useful -- it is a popular OS, and does do some things differently from other Unixes.